pageserver/src/layered_repository/image_layer.rs

```diff
@@ -279,7 +279,6 @@ impl ImageLayer {
             // above call to `load_inner`, so it's already been released). And
             // while we do that, another thread could unload again, so we have
             // to re-check and retry if that happens.
-            drop(inner);
```

Comment:
I'd prefer to keep the `drop(inner)`. It's not necessary because the scope ends here anyway, but I think it's more clear to be explicit about it, since the comment talks about it too.

Reply:
I just want to make image_layer.rs code similar with delta_layer.rs